deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.2/cdi.crd.yaml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
The path should now be `deploy/olm-catalog/kubevirt-hyperconverged/0.0.2/...`, maybe you need to rebase and rebuild the manifests?

Reply:
I think this is leftovers..

Reply:
Looks like you have both. Maybe just remove `deploy/converged`.

Reply:
I thought the rebase was too easy :).  Removed.